src/main/java/com/project/model/Application.java

```diff
@@ -4,6 +4,7 @@
 import lombok.EqualsAndHashCode;
 import lombok.NoArgsConstructor;
 import lombok.ToString;
+
 import javax.persistence.*;
```

Comment:
You should not import everything from a package if you do not need it. Better to just import what you need.

Reply:
Intellij automatically changes imports to import package.* when there are more than 4 elements imported from the same package. As there are some classes with more than 8 or 10 imports from the same package, i think it is better to let intellij handle the imports as writing them manually every time you add a new class to import from the same package is going to be time consuming